static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -193,9 +203,43 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
     searchThisAreaButtonEl.addEventListener('click', (e) => {
       this.searchThisArea();
     });
-
     this.setupMobileBreakpointListener();
     this.addMapComponent();
+    this.setFixedHeightsOnAndroid()
+  }
+
+  /**
+   * On Android browsers, opening up the keyboard will shift the contents of the entire page up,
+   * moving the map center, and thereby causing a searchOnMapMove to be triggered.
+   * The search response would then cause the page to update,
+   * and close the keyboard, making it impossible to actually type anything into the searchbar.
+   * 
+   * Setting a fixed height on elements like .Answers-mapWrapper prevents the keyboard from shifting the content
+   * of the page.
+   */
+   setFixedHeightsOnAndroid() {
+    if (!this.isMobile() || !/Android/i.test(navigator.userAgent)) {
+      return
+    }
+
+    setFixedHeight('.Answers-mapWrapper')
+
+    function getSingleElement(selector) {
+      const els = document.querySelectorAll(selector)
+      if (els.length === 0) {
+        console.error(`No ${selector} found, unable to set fixed height.`)
+      } else if (els.length > 1) {
+        console.error(`Multiple elements of for ${selector} found, expected only 1, not setting fixed height.`)
```

Comment:
The issue still occurs with multiple map elements on page right? Can we not set fixed height for all even if there are multiple map wrapper elements found?

Reply:
The full page map should never have multiple maps. If you're doing something custom with the map I didn't want to make assumptions